### DISCUSSION TITLE
Detecting suspicious windows behavior

### DISCUSSION THREAD

**Discussion by OlexTratisky** [Security-Onion-Solutions/securityonion#9409] [General] [answered]
Hello community,

Is there any specific plugin or feature that analyses windows event log and tries to correlate if there is any suspicious activity?  For example, try to detect priviledge escalation, brute force attempts, file deletion of critical files, exfiltration etc.

I am currently adding the winlogbeat to SO but cant see any alarms for windows events that might be attacks

Thank you
Olex

**Comment by dougburks** [comment, marked as answer]
You can use [Playbook](https://docs.securityonion.net/en/2.3/playbook.html) to enable [Sigma rules](https://github.com/SigmaHQ/sigma/tree/master/rules/windows) or write your own detections.
